tests/e2e/tidbcluster/serial.go

```diff
@@ -351,6 +352,18 @@ var _ = ginkgo.Describe("[tidb-operator][Serial]", func() {
 			oa.CheckTidbClusterStatusOrDie(&cluster)
 			oa.CleanTidbClusterOrDie(&cluster)
 		})
+
+		ginkgo.It("[PodAdmissionWebhook] delete pvc before deleting pod directly with admission webhook", func() {
```

Comment:
what is the purpose of this case?

Reply:
https://github.com/pingcap/tidb-operator/issues/1567
In this issue, we found that the admission webhook sever denied the deleting request because of failure to find its `pvc` during scale-in.
We think the request should be permitted during scale-in when `pvc` is not found.

Reply:
I also will add this case into e2e test to cover it.

Reply:
revert this unfinished change in this PR? we can submit a separate PR to address https://github.com/pingcap/tidb-operator/issues/1569 if necessary.